sw/device/lib/dif/dif_clkmgr_unittest.cc

```diff
@@ -22,6 +22,57 @@ class ClkMgrTest : public Test, public MmioTest {
   dif_clkmgr_t clkmgr_ = {.base_addr = dev().region()};
 };
 
+class JitterEnableTest : public ClkMgrTest {};
+
+// SetEnabled uses EXPECT_WRITE32 instead of EXPECT_MASK32 because
+// dif_clkmgr_jitter_set_enabled doesn't perform a read, just a write.
+TEST_F(JitterEnableTest, SetEnabled) {
+  // Disable jitter.
+  EXPECT_WRITE32(CLKMGR_JITTER_ENABLE_REG_OFFSET, 0);
+  EXPECT_EQ(dif_clkmgr_jitter_set_enabled(&clkmgr_, kDifToggleDisabled),
+            kDifOk);
+
+  // Enable jitter.
+  EXPECT_WRITE32(CLKMGR_JITTER_ENABLE_REG_OFFSET, 1);
+  EXPECT_EQ(dif_clkmgr_jitter_set_enabled(&clkmgr_, kDifToggleEnabled), kDifOk);
+}
+
+TEST_F(JitterEnableTest, SetEnabledError) {
+  // Null handle.
+  EXPECT_EQ(dif_clkmgr_jitter_set_enabled(nullptr, kDifToggleEnabled),
+            kDifBadArg);
+}
+
+TEST_F(JitterEnableTest, GetEnabled) {
+  // Get jitter (enabled).
+  {
+    bool enabled = false;
+    EXPECT_READ32(CLKMGR_JITTER_ENABLE_REG_OFFSET,
+                  {{CLKMGR_JITTER_ENABLE_VAL_BIT, true}});
+    EXPECT_EQ(dif_clkmgr_jitter_get_enabled(&clkmgr_, &enabled), kDifOk);
+    EXPECT_TRUE(enabled);
+  }
+
+  // Get jitter (disabled).
+  {
+    bool enabled = true;
+    EXPECT_READ32(CLKMGR_JITTER_ENABLE_REG_OFFSET,
+                  {{CLKMGR_JITTER_ENABLE_VAL_BIT, false}});
+    EXPECT_EQ(dif_clkmgr_jitter_get_enabled(&clkmgr_, &enabled), kDifOk);
+    EXPECT_FALSE(enabled);
+  }
+}
+
+TEST_F(JitterEnableTest, GetEnabledError) {
+  bool enabled = false;
+
+  // Null handle.
```

Comment:
I don't think you need these comments... the code below is self-explanatory.

Reply:
Gone